.github/workflows/CI.yml

```diff
@@ -12,20 +12,29 @@ jobs:
       matrix:
         os: [windows-latest, ubuntu-latest]
     runs-on: ${{matrix.os}}
-    env:
-      buildVersion: 0.0.0.0 #The initial build version, calculated and updated in the script below
-      ACTIONS_ALLOW_UNSECURE_COMMANDS: true #Allow us to set an environment variable in the PowerShell script to set the package version (technically deprecated)
     steps:
-    - uses: actions/checkout@v1
-    - name: Custom script to create a unique application version number for this build
-      run: src/BuildVersion.ps1 -ProjectFile "src/AzurePipelinesToGitHubActionsConverter.Core/AzurePipelinesToGitHubActionsConverter.Core.csproj"
-      shell: pwsh
-    - name: Build with dotnet
-      run: dotnet build src/AzurePipelinesToGitHubActionsConverter.Core/AzurePipelinesToGitHubActionsConverter.Core.csproj --configuration Release
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 #fetch-depth is needed for GitVersion
```

Comment:
yes complete fetchDepth is required for gitversion, but that happens by default, so you can delete lines 17 & 18